e2e/tests/user/login.spec.ts

```diff
@@ -18,7 +18,7 @@ describe('Login tests:', () => {
   });
 
   test('Open AoU Workspaces page before login redirects to login page', async () => {
-    const url = `${config.uiBaseUrl}${config.workspacesUrlPath}`;
+    const url = `${config.LOGIN_URL}${config.workspacesUrlPath}`;
```

Comment:
why are some config values all caps and others lowercase?

Reply:
Changed to all uppercase for consistency.